traits/ctraits.c

```diff
@@ -4519,6 +4519,23 @@ _get_trait_comparison_mode_int(trait_object *trait, void *closure)
     return PyLong_FromLong(i_comparison_mode);
 }
 
+/*-----------------------------------------------------------------------------
+|  Get the 'property' value fields of a CTrait instance:
++----------------------------------------------------------------------------*/
+
+static PyObject *
+_get_property(trait_object *trait, PyObject *args)
+{
+    if (trait->flags & TRAIT_PROPERTY) {
```

Comment:
`_get_property` should raise `TypeError` if any arguments are passed to it, so we should be checking `args` somehow. It's possible to do that in this function, but a cleaner way is to use the `METH_NOARGS` passing convention. See `_trait_get_validate` for an example of how to do that.

We probably also want a unit test that verifies that a `TypeError` is raised if any arguments are passed. Currently, any arguments passed by position are ignored.

Reply:
Used `METH_NOARGS` and added a test for `_get_property` with arguments.